Car_Collision_Visulaization/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+cmake_minimum_required(VERSION 3.0.0)
+project(DataVisualization VERSION 0.1.0 LANGUAGES C CXX)
+
+
+add_library(
+    Visualizer
+    src/CImg.h 
+    src/data_visualization.h
+    src/data_visualization.cpp
+)
+
+add_library(
+    Car
+    src/car_collision.h
+    src/car_collision.cpp
+)
```

Comment:
car_collision.h and car_collision.cpp
are not meaningful names anymore please change it